code/game/machinery/firealarm.dm

```diff
@@ -62,78 +62,74 @@ FIRE ALARM
 /obj/machinery/firealarm/attack_ghost(mob/user)
 	ui_interact(user)
 
-/obj/machinery/firealarm/bullet_act(BLAH)
```

Comment:
Why did you remove this?

Reply:
Having custom overridden behaviors for bullet_act, and the likes should be avoided in the vast majority of instances; we want the item to be damaged, not for this to happen.

I suspect this was put in, by Bay, on the assumption of "lol, lasers are hot, it would trigger the fire alarm". It's an annoying little behavior that doesn't really fit realistically, and is going to get in the way of future behaviors (I don't think having this behavior *and* damaging it, simultaneously works well, either).

Reply:
Well you could argue with the temperature gun that it could fuck with air alarms, but fire alarms are just switches, not sensors correct? Isn't it the air alarm that triggers containment and the fire alarm that disarms it before optimal atmos returns?

Be kind of neat to trigger alarms with matches or lighters without blowing stuff up

Reply:
They work off temperature; if the temperature is too high, then they will alarm.